src/main.cpp

```diff
@@ -55,6 +56,7 @@ int main(int argc, char** argv) {
     Barrier barrier = EPOCH;
 
     bool reindex = false;
+    std::string reindex_of = "project";
```

Comment:
you need to add this attribute to the config.ini.template 
and also we need to propagate this information from elasticsearch-chef and epipe-chef
https://github.com/logicalclocks/elasticsearch-chef/blob/master/attributes/default.rb#L48
https://github.com/logicalclocks/epipe-chef/blob/master/recipes/default.rb#L67

Reply:
fixed